### ISSUE TITLE
Switch to plain eslint config

### ISSUE THREAD

**Issue by dinoboff** [alessioalex/git-commits#4]
It would easier for contributors to have a plain eslint setting file. Even if there was an alessioalex-standard plugin for my IDE I wouldn't want to install one just for one odd project.

**Comment by alessioalex** [comment]
That's true, I'll just add some plain eslint config soon. Meanwhile there's a git hook that'll let you know of any linting issues.
